src/olympia/addons/management/commands/fix_langpacks_with_max_version_star.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import BaseCommand
+
+from olympia import amo
+from olympia.core.logger import getLogger
+from olympia.versions.compare import version_dict
+from olympia.versions.models import AppVersion, Version
+
+log = getLogger('z.fix_langpacks_with_max_version_star')
+
+
+class Command(BaseCommand):
+    help = 'Fix language packs that have a max version compatibility set to *'
+
+    def find_affected_langpacks(self):
+        qs = Version.unfiltered.filter(
+            addon__type=amo.ADDON_LPAPP, apps__max__version='*').distinct()
+        return qs
+
+    def fix_max_appversion_for_version(self, version):
+        for app in (amo.FIREFOX, amo.ANDROID):
+            if app not in version.compatible_apps:
+                log.info(
+                    'Version %s for addon %s min version is not compatible '
+                    'with %s, skipping this version for that app.',
+                    version, version.addon, app.pretty)
```

Comment:
this is going to be a pretty noisy log

Reply:
Yeah - I figured we wanted to be extra sure we wouldn't be missing anything, and if we were what the reason was